src/vm.c

```diff
@@ -164,6 +164,8 @@ TJSRuntime *TJS_NewRuntimeWorker(void) {
 }
 
 TJSRuntime *TJS_NewRuntimeInternal(bool is_worker, TJSRunOptions *options) {
+    if(tjs_process_start_uptime == -1) 
```

Comment:
Maybe better use a uv_once for this, since there could be multiple threads started with different runtimes.